src/components/button/index.js

```diff
@@ -1,7 +1,16 @@
 import React from "react";
+import "./css.module.scss";
 import * as css from "./css.module.scss";
 
 const Button = (props) => {
-  return <button className={css.circular} onClick = {props.onClick}>{props.children}</button>;
+  //const styles = []
+    return (
+    <div>
+      <button className = {css.circular} id = {props.id} onClick = {props.onClick}>{props.children}</button>
```

Comment:
You can do the following on the class declaration
`<Button type="roundedEdges" />`
For the component itself you can now do
`<button className={css[props.type]} />`

Reply:
Woah! I was able to transform like 6 if statements into two conditionals lol maaan. I was looking for something like this before because I didn't want to write so many conditionals, but couldn't find anything. Thanks a million for this